src/services/io/podio/JEventProcessorPODIO.h

```diff
@@ -32,4 +32,6 @@ class JEventProcessorPODIO : public JEventProcessor {
     std::vector<std::string> m_collections_to_write;  // derived from above config. parameters
     std::vector<std::string> m_collections_to_print;
 
+  private:
+    std::set<std::string> m_failed_collections;
```

Comment:
I've submitted a few PRs for HcalEndcapP. With those merged, we could fail immediately instead of just making warnings.